code/python/raadpleeg_met_bsn.py

```diff
@@ -0,0 +1,28 @@
+import requests
```

Comment:
ooit was code bedoeld om alvast gegenereerde clients te geven. En daarbij (impliciet?) te controleren dat het genereren goed lukte uit de OAS  specificaties.

Dit is denk ik iets anders. Waar komt dit vandaan en waarvoor is dit nodig?